src/infra/core/src/testing/dataset_data_helper.rs

```diff
@@ -93,6 +110,18 @@ impl DatasetDataHelper {
             .unwrap()
     }
 
+    pub async fn get_last_schema(&self) -> SchemaRef {
```

Comment:
Maybe "latest", "relevant".

Reply:
renamed to `get_latest_set_schema` to refer to the event type more clearly (not to be confused with retrieving a schema from Parquet file)